main.go

```diff
@@ -24,17 +24,19 @@ import (
 
 	"path/filepath"
 
+	"encoding/json"
+
 	"github.com/dongmx/rdb"
-	"github.com/elazarl/go-bindata-assetfs"
+	assetfs "github.com/elazarl/go-bindata-assetfs"
 	"github.com/julienschmidt/httprouter"
-	"github.com/xueqiu/rdr/static"
-	"encoding/json"
+	"github.com/YongHaoWu/rdr/decoder"
+	"github.com/YongHaoWu/rdr/static"
```

Comment:
This import should be `github.com/xueqiu/rdr/static`

Reply:
@dongmx Yeah, I will fix it later. I just need to import it in my fork.. But..as the PR is from my fork's master to your master, so whatever I push to master will show on this pr

Reply:
@dongmx I will create a new pr